lib/livebook/hubs/dockerfile.ex

```
@@ -157,6 +157,46 @@ defmodule Livebook.Hubs.Dockerfile do
     |> Enum.join("\n")
   end
 
+  @doc """
+  Builds Dockerfile definition for Livebook Agent app deployment.
+  """
+  @spec build_agent_dockerfile(config(), Hubs.Provider.t()) :: String.t()
+  def build_agent_dockerfile(config, hub) do
+    base_image = Enum.find(Livebook.Config.docker_images(), &(&1.tag == config.docker_tag))
+
+    image = """
+    FROM ghcr.io/livebook-dev/livebook:#{base_image.tag}
+    """
+
+    image_envs = format_envs(base_image.env)
+
+    hub_config = """
+    ARG TEAMS_KEY="#{hub.teams_key}"
+    ARG AGENT_KEY=""
+    ARG AGENT_NAME="put-your-custom-agent-name"
```

We can add a note on the custom agent name below the dockerfile, similar to how we do with the other Dockerfile :) By not putting it here Docker is going to fail if they don't specify the arguments

```suggestion
    ARG AGENT_KEY
    ARG AGENT_NAME
```